### PR TITLE
doc: update supported devices list

### PR DESCRIPTION
**Pull request by greg-fer**
Updated the list of supported devices.
Fixed a typo in changelog.
NCS-698.

### CODE REVIEW

**Comment by boundlesscalm** [comment]
Is the rest of the latest changelog block good to go also?

**Comment by greg-fer** [comment]
> Is the rest of the latest changelog block good to go also?

Now it is.

File: Changelog.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 
--   Improved error/notice UI.
+-   Improved error and notice UI.
+-   The Program step, which now ensures that the device is connected before programming.
 
 ### Fixed
 
--   Reading the ICCID value sometimes never finished.
--   Incorrect button label when failing to program.
+-   An issue with reading the ICCID value that would never finish.
```
Comment (boundlesscalm) [inline Changelog.md:15]: This sounds confusing.
"An issue ... that would never finish"

File: Changelog.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 
 ### Added
 
--   `SIM Card` step for nRF9160 DK.
+-   The SIM Card step for nRF9160 DK.
 
 ### Fixed
 
--   Failed to list devices sometimes.
+-   An rare issue with failing to list devices.
```
Comment (boundlesscalm) [inline Changelog.md:26]:
```suggestion
-   A rare issue with failing to list devices.
```

File: Changelog.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
--   Improved error/notice UI.
+-   Improved error and notice UI.
+-   The Program step, which now ensures that the device is connected before programming.
 
 ### Fixed
 
--   Reading the ICCID value sometimes never finished.
--   Incorrect button label when failing to program.
+-   An issue with reading the ICCID value that would never finish.
+-   The incorrect button label when failing to program.
```
Comment (boundlesscalm) [inline Changelog.md:16]:
```suggestion
-   An incorrect button label when failing to program.
```

**Review by boundlesscalm — COMMENTED** [review]
Let's wait until we decide on general language to merge these changes.

File: Changelog.md (hunk 1 above)
Comment (greg-fer) [reply Changelog.md]: No, "reading the ICCID value that would never finish".
This is the style that was adopted in release notes since nRF5 SDK.

**Comment by greg-fer** [comment]
> Let's wait until we decide on general language to merge these changes.

@boundlesscalm , I've reverted the changes as the original scope of the PR is to update docs and fix the typo in the changelog.

If we need to discuss how changelog is written, let's do it, but I want to update docs as soon as possible with the new support info.

**Comment by boundlesscalm** [comment]
Sounds good, just merge it.

**Pull request merged**